packages/sdk/src/PreVerificationGasCalculator.ts

```diff
@@ -1,31 +1,57 @@
 import { encodeUserOp, UserOperation } from '@account-abstraction/utils'
-import { arrayify, hexlify } from 'ethers/lib/utils'
+import { arrayify, hexDataLength, hexlify } from 'ethers/lib/utils'
+import { bytesToHex } from '@ethereumjs/util'
+import { BigNumber, BytesLike } from 'ethers'
+
+const EXECUTE_USEROP_METHOD_SIG = '0x8dd7712f'
 
 export interface PreVerificationGasCalculatorConfig {
   /**
    * Cost of sending a basic transaction on the current chain.
    */
   readonly transactionGasStipend: number
   /**
-   * Gas overhead is added to entire 'handleOp' bundle.
+   * Gas overhead is added to entire 'handleOp' bundle (on top of the transactionGasStipend).
    */
   readonly fixedGasOverhead: number
   /**
    * Gas overhead per UserOperation is added on top of the above fixed per-bundle.
    */
   readonly perUserOpGasOverhead: number
+
   /**
-   * Gas overhead per single "word" (32 bytes) of an ABI-encoding of the UserOperation.
+   * Gas overhead per single "word" (32 bytes) in callData.
+   * (all validation fields are covered by verification gas checks)
    */
   readonly perUserOpWordGasOverhead: number
   /**
-   * The gas cost of a single zero byte an ABI-encoding of the UserOperation.
+   * extra per-userop overhead, if callData starts with "executeUserOp" method signature.
+   */
+  readonly execUserOpGasOverhead: number
+  /**
+   * extra per-word overhead, if callData starts with "executeUserOp" method signature.
+   */
+  readonly execUserOpPerWordGasOverhead: number
```

Comment:
```suggestion
  readonly executeUserOpPerWordGasOverhead: number
```

Reply:
You saved 3 characters and you will never be able to grep it... Not a good deal.